dotnet/src/Connectors/Connectors.Memory.AzureCosmosDBMongoDB/AzureCosmosDBMongoDBMemoryRecordMetadata.cs

```diff
@@ -0,0 +1,80 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using Microsoft.SemanticKernel.Memory;
+using MongoDB.Bson.Serialization.Attributes;
+
+namespace Microsoft.SemanticKernel.Connectors.AzureCosmosDBMongoDB;
+
+/// <summary>
+/// A MongoDB memory record metadata.
+/// </summary>
+#pragma warning disable CA1815 // Override equals and operator equals on value types
+public struct AzureCosmosDBMongoDBMemoryRecordMetadata
+#pragma warning restore CA1815 // Override equals and operator equals on value types
+{
+    /// <summary>
+    /// <see cref="AzureCosmosDBMongoDBMemoryRecordMetadata.IsReference"/>.
```

Comment:
Looks like these comments are just pointing to the property instead of explaining what the property is for.

Reply:
Updated the comments